### ISSUE TITLE
希望能添加更多的lyrics server搜索来源

### ISSUE THREAD

**Issue by imuqie** [MartianZ/DynamicLyrics#59]
现在的歌词的来源有点不够用 貌似只有从国内网络上的歌词才会被搜索到
很多外语歌只能看着文不对题的歌词 搜索也搜索不到
而且iTunes上本来有些歌本身就带lyrics的也不能显示本地歌词
希望能考虑一下添加搜索源 :))))

**Comment by speakerz** [comment]
同感

**Comment by cogitoxin** [comment]
Agree, like this one https://github.com/iampennyli/iTunesLyricApp fetching lyrics from 网易云。
